sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/implementation/AzureConfigurableImpl.java

```diff
@@ -36,6 +36,7 @@
     private RetryPolicy retryPolicy;
     private Configuration configuration;
     private List<TokenCredential> tokens;
+    private HttpPipeline internalHttpPipeline;
```

Comment:
nit, maybe `preconfiguredHttpPipeline` would be a better name.

Not important at this stage.